xformers/helpers/test_utils.py

```diff
@@ -4,10 +4,15 @@
 # LICENSE file in the root directory of this source tree.
 
 
+import os
 import tempfile
 
 import torch
 
+is_windows = False
+if os.environ.get("OS", "") == "Windows_NT":  # pytorch on windows uses gloo not ncll
+    is_windows = True
```

Comment:
Would a more robust alternative be to use
```python
sys.platform == 'win32'
```
?

This is what is used in PyTorch in many places [1](https://github.com/pytorch/pytorch/blob/08780229df8f860dbef3fa82ffd1072b124b29c5/torch/utils/cpp_extension.py#L27) [2](https://github.com/pytorch/pytorch/blob/c371542efc31b1abfe6f388042aa3ab0cef935f2/torch/multiprocessing/_atfork.py#L5)
plus, this might also enable other system that are not Windows_NT (7, XP, etc?)

Reply:
change made as suggested